configs/base/packages.yaml

```diff
@@ -1,46 +1,35 @@
 packages:
   ascent:
-    variants: ~fortran~openmp
+    require: "~fortran~openmp"
   exawind:
-    version: [master]
+    require: "@master"
   nalu-wind:
-    version: [master]
+    require: "@master ~trilinos-solvers"
```

Comment:
I don't think this should be a requirement. Too strict to require versions and variants of our own software unless we know it won't build on a machine.

Reply:
The default is `@multiphase` without this.

Reply:
Oh, or you mean keep it as a preference?

Reply:
Yeah or re order them. I know there is a way to set the default branch